README.md

```diff
@@ -1,2 +1,9 @@
 # kddeployqt
-Deployment tool for Qt applications, similar to (android/mac/win)deployqt
+Deployment tool for Qt applications, similar to (android/mac/win)deployqt.
+
+The main differences between this tool and the existing ones([linuxdeploqt](https://github.com/probonopd/linuxdeployqt), and [linuxdeploy](https://github.com/probonopd/linuxdeploy)) are:
```

Comment:
I think the current, most popular tool named "linuxdeploy" is this: https://github.com/linuxdeploy/linuxdeploy